### PR TITLE
add HTML & CSS practice: Hooli-style Popup

### PR DESCRIPTION
**Pull request by KikinovK**
### Html-css-popup

[Demo](https://kikinovk.github.io/frontend-2021-homeworks/submissions/kikinovk/HTML_CSS_Popup/) |
[Code base](https://github.com/KikinovK/frontend-2021-homeworks/tree/html-css-popup-task/submissions/kikinovk/HTML_CSS_Popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/kikinovk/HTML_CSS_Popup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+  outline:none;
+}
+html {
+  height: 100%;
+}
+body {
+  display: grid;
+  min-height: 100%;
+  grid-template-rows: auto 1fr;
+  font-family: sans-serif;;
```
Comment (lenadgit) [inline submissions/kikinovk/HTML_CSS_Popup/css/style.css:13]: could you please remove a second semicolon

File: submissions/kikinovk/HTML_CSS_Popup/index.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+        </a>
+      </li>
+      <li class="menu__item">
+        <input 
+          type="checkbox"
+          name="toggle_pop_up"
+          id="toggle_pop_up"
+          class="toggle__input" 
+        >
+        <Label
```
Comment (lenadgit) [inline submissions/kikinovk/HTML_CSS_Popup/index.html:30]: could you please replace a capital letter

File: submissions/kikinovk/HTML_CSS_Popup/index.html [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+          <span class="menu__icon icon--user"></span>
+        </a>
+      </li>
+    </ul>
+  </nav>
+  <main class="main">
+    
+  </main>
+</body>
+</html>
```
Comment (lenadgit) [inline submissions/kikinovk/HTML_CSS_Popup/index.html:174]: could please add an empty line

File: submissions/kikinovk/HTML_CSS_Popup/css/style.css [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+  width: 100%;
+  text-align: center;
+  cursor: pointer;
+  background-color: rgba(102, 102, 102, 0.2);
+}
+.button_more__input:hover + .button_more__label, 
+.button_more__input:focus + .button_more__label{
+  color: #0033cc;
+  text-decoration: underline;
+}
```
Comment (lenadgit) [inline submissions/kikinovk/HTML_CSS_Popup/css/style.css:265]: could you please add an empty line

**Comment by lenadgit** [comment]
Excellent job in my opinion!

**Review by MaxKalashnyk — CHANGES_REQUESTED** [review]
Good job! Please fix all errors of your preview in the [validator](https://validator.w3.org/)

**Comment by KikinovK** [comment]
> The empty line must be after the last tag/symbol. It s good practice https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file Could you please fix?

**Review by lenadgit — APPROVED** [review]

File: submissions/kikinovk/HTML_CSS_Popup/css/style.css [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+.menu__link {  
+  text-decoration: none;
+  color: #666666;
+}
+.menu__link:hover, .menu__link:focus{
+  color: #0033cc;
+  text-decoration: underline;
+}
+.menu__link:focus > .icon--bell {
+  background-image: url(../image/bell-hover.png);
```
Comment (MaxKalashnyk) [inline submissions/kikinovk/HTML_CSS_Popup/css/style.css:48]: Changing different images by using `background-image` on each hover or focus can drastically decrease the performance of your styles. Better to use SVG icons here or some font icons and just toggle `fill` or `stroke` properties (for the SVGs) or color (for the font icons) on hover/focus/default states.
Let's leave these icons. But don't forget to use the SVG or font icons in the future, please

**Review by MaxKalashnyk — APPROVED** [review]

**Pull request merged**